bower.json

```diff
@@ -9,11 +9,13 @@
   "license": "MIT",
   "private": true,
   "dependencies": {
+    "baguettebox.js": "~1.8",
     "bootstrap": "~3.3.7",
+    "html5shiv": "^3.7.3",
     "jquery": "<2.0.0",
     "jquery-colorbox": "~1",
-    "moment": "~2",
+    "katex": "^0.7.1",
```

Comment:
What is the reason we "can't afford to ship KaTeX"?

If I look at what we're already loading via bower, it's less than half the size of moment and roughly comparable to bootstrap. MathJax is >100MB but KaTeX is relatively small.

```
$ du -sh *
112K	baguettebox.js
2.5M	bootstrap
 88K	html5shiv
1.2M	jquery
360K	jquery-colorbox
2.6M	katex
 72K	livereload-js
6.5M	moment
```

Maybe the thing to do is open an issue to create a task for asset retrieval and not store them in the repo at all.